### PR TITLE
Update to newest RTC Engine

### PR DESCRIPTION
**Pull request by daniel-jodlos**

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -28,7 +28,7 @@ defmodule VideoRoom.MixProject do
 
   defp deps do
     [
-      {:membrane_rtc_engine, "~> 0.8.0"},
+      {:membrane_rtc_engine, github: "jellyfish-dev/membrane_rtc_engine", override: true},
       {:membrane_rtp_plugin,
```
Comment (bblaszkow06) [inline mix.exs:32]: Please update this plugin as well.

BTW, you can review the last added commit: https://github.com/membraneframework/membrane_rtp_plugin/commit/aa197acda401fb282a5c5be8d11acc45c4575212

File: lib/videoroom/room.ex [hunk 2]
```diff
@@ -210,15 +185,34 @@ defmodule Videoroom.Room do
...
+  end
+
+  defp filter_codecs({%{encoding: "H264"}, fmtp}) do
+    import Bitwise
 
-      {:noreply, state}
+    case fmtp.profile_level_id >>> 16 do
+      0x42 -> (fmtp.profile_level_id &&& 0x00_4F_00) == 0x00_40_00
+      0x4D -> (fmtp.profile_level_id &&& 0x00_8F_00) == 0x00_80_00
+      0x58 -> (fmtp.profile_level_id &&& 0x00_CF_00) == 0x00_C0_00
```
Comment (bblaszkow06) [inline lib/videoroom/room.ex:208]: I'd would be nice to add a comment on that - what profiles are we selecting here?

**Review by bblaszkow06 — APPROVED** [review]

File: lib/videoroom/room.ex [hunk 3]
```diff
@@ -45,7 +44,8 @@ defmodule Videoroom.Room do
     rtc_engine_options = [
       id: room_id,
       trace_ctx: trace_ctx,
-      parent_span: room_span
+      parent_span: room_span,
+      toilet_capacity: 1000
```
Comment (mickel8) [inline lib/videoroom/room.ex:48]: toilet_capacity has to be set in WebRTC endpoint as TR works in push/push mode

File: lib/videoroom/room.ex [hunk 4]
```diff
@@ -179,22 +161,15 @@ defmodule Videoroom.Room do
...
 
-    error_message = "WebRTC endpoint has crashed, please refresh the page to reconnect"
-
-    data =
-      error_message
-      |> MediaEvent.create_error_event()
-      |> MediaEvent.encode()
-
-    send(peer_channel, {:media_event, data})
+    GenServer.stop(peer_channel, :endpoint_crashed)
```
Comment (mickel8) [inline lib/videoroom/room.ex:164]: Does it mean that we won't inform FE about endpoint crash? The only thing FE will notice is that WS connection was closed :thinking:

**Review by mickel8 — CHANGES_REQUESTED** [review]
One more thing is taht dscout targets 1280x720 and I am not sure whether we want to target it too but I think we should? cc @bblaszkow06

File: lib/videoroom/room.ex (hunk 4 above)
Comment (daniel-jodlos) [reply lib/videoroom/room.ex:164]: Yes. But it should now that it didn't close cleanly. Also, `error` media event doesn't exist anymore https://github.com/jellyfish-dev/membrane_rtc_engine/pull/214#discussion_r1062436369. But we probably need to make sure that the FE handles phoenix channel errors correctly

**Review by mickel8 — APPROVED** [review]

**Pull request merged**